app/controllers/participants_controller.rb

```diff
@@ -13,18 +16,35 @@ def show
     end
     @personal_questionnaire = PersonalQuestionnaireSerializer.new(personal_q)
 
-    
     if @participant.professional_questionnaire.nil?
       professional_q = ProfessionalQuestionnaire.create("participant_id": @participant.id)
     else
       professional_q = @participant.professional_questionnaire
     end
     @professional_questionnaire = ProfessionalQuestionnairesSerializer.new(professional_q)
-
     @studio_assessments = @participant.studio_assessments
+
+    @assignment_list = []
+    @assignments.each do |a|
+      action_item = ActionItem.where(id: a.action_item_id).first
```

Comment:
why do you create new objects instead of sending the already created objects?

Reply:
I think for the assignments I get from the back-end, they're not completely constructed (they have the different instance attributes for that participant's actionItem). So, I have to find the other parts of the ActionItem (title, desc, category, is_template) from the ActionItem object and construct new "complete assignments" to be passed to the front-end

Reply:
ah got it, sounds good